### PR TITLE
iPad views are here

### PR DESCRIPTION
**Pull request by yousefhamza**
This patch includes commits to make the app iPad ready 100% :)

### CODE REVIEW

File: OpenMRS-iOS/MainMenuCollectionViewController.m [hunk 1]
```diff
@@ -58,6 +64,9 @@ - (NSInteger)numberOfSectionsInCollectionView:(UICollectionView *)collectionView
 
 - (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSection:(NSInteger)section
 {
+    if (UI_USER_INTERFACE_IDIOM() == UI_USER_INTERFACE_IDIOM()) {
```
Comment (Undo1) [inline OpenMRS-iOS/MainMenuCollectionViewController.m]: What's this for? Seems like it'd always be true :)

**Comment by harshadura** [comment]
good job, im pretty busy these days. will have a look when i get some time. @Undo1 would you mind giving feedback about @yousefhamza work on this, thanks!

**Comment by yousefhamza** [comment]
@Undo1 I fixed that line.

**Comment by Undo1** [comment]
Looks good, I made a few changes. See 1c3e8d59c2277f35cd4299f3a7e45d51c61aac64 and 90e088482075838aa212a7eb365860994a4a7286

**Pull request merged**